### PR TITLE
add support for multiple files using ffsend

### PR DESCRIPTION
**Pull request by Darukutsu**
Maybe not needed since you can archive, compress, send and then delete archive on your own. This ffsend feature only archives without compression.

### CODE REVIEW

File: plugins/upload [hunk 1]
```diff
@@ -4,27 +4,40 @@
...
-        curl -F "f:1=@$1" ix.io
+      # File name will be randomized foo.tar
+      xargs -0 < "$selection" ffsend u
     else
-        # Upload the file, show the download link and wait till user presses any key
-        curl -s -F "file=@$1" https://file.io/?expires=1w | jq '.link' | tr -d '"'
-
-        # To write download link to "$1".loc and exit
-        # curl -s -F "file=@$1" https://file.io/?expires=1w -o `basename "$1"`.loc
+      printf "For sending multiple(marked) files, ffsend is required."
```
Comment (jarun) [inline plugins/upload:20]: Please change to:

    ffsend is required to upload selection.

File: plugins/upload [hunk 2]
```diff
@@ -4,27 +4,40 @@
...
+# Dependencies: ffsend (https://github.com/timvisee/ffsend), curl, jq, tr, sed
 #
 # Note: Binary file set to expire after a week
 #
 # Shell: POSIX compliant
 # Author: Arun Prakash Jana
 
-if [ -n "$1" ] && [ -s "$1" ]; then
+selection=${NNN_SEL:-${XDG_CONFIG_HOME:-$HOME/.config}/nnn/.selection}
+if [ -s "$selection" ]; then
```
Comment (jarun) [inline plugins/upload:15]: Please update the documentation that selection is handled exclusively using `ffsend`.

**Review by jarun — CHANGES_REQUESTED** [review]

File: plugins/upload [hunk 3]
```diff
@@ -4,9 +4,10 @@
 #              Paste contents of a text a file http://ix.io
 #              Upload a binary file to file.io
 #
-# Dependencies: ffsend (https://github.com/timvisee/ffsend), curl, jq, tr, sed
+# Dependencies: ffsend (https://github.com/timvisee/ffsend), curl, jq, tr
 #
 # Note: Binary file set to expire after a week
+# 	Sending multiple files at once is handled exclusively using ffsend.
```
Comment (jarun) [inline plugins/upload:10]: Sending selection is handled exclusively using ffsend.

**Review by jarun — CHANGES_REQUESTED** [review]

File: plugins/upload (hunk 3 above)
Comment (Darukutsu) [reply plugins/upload]: Feel free to edit I permit edit access. Sorry github-ui resolve commits are weird IDK how to operate this.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**